src/app/lib/config/optimizely/index.js

```diff
@@ -1,5 +1,14 @@
 export default {
-  experimentId: '004_brasil_recommendations_experiment',
-  featureId: '004_brasil_recommendations_experiment',
+  experimentId: '005_brasil_hybrid_recommendations_',
+  featureId: '005_brasil_hybrid_recommendations_',
   viewClickAttributeId: 'wsoj',
+  variationMappings: {
+    content_recs: 'datalabContentRecommendations',
```

Comment:
Similar to above, I think this is from the old experiment, so may not be needed.